src/components/Home/index.css

```diff
@@ -1,69 +1,28 @@
+.flex_container {
+  display: flex;
+  flex-direction: row;
```

Comment:
you don't have to specify the flex direction here since `row` is the default value 
can be deleted